cvat/apps/dataset_manager/util.py

```diff
@@ -80,3 +92,140 @@ def deepcopy_simple(v):
         return v
     else:
         return deepcopy(v)
+
+
+class LockNotAvailableError(Exception):
+    pass
+
+
+def make_export_cache_lock_key(filename: os.PathLike[str]) -> str:
+    return f"export_lock:{filename}"
```

Comment:
```suggestion
    return f"export_lock:{os.fspath(filename)}"
```

There's no requirement for `str` to return anything useful for `PathLike`.